crates/cxx-qt-gen/src/generator/cpp/qobject.rs

```diff
@@ -118,6 +121,8 @@ impl GeneratedCppQObject {
             .insert("#include <cxx-qt-common/cxxqt_maybelockguard.h>".to_owned());
 
         // Build the base class
+        //
+        // TODO: if there is no qobject macro should we assume the base?
```

Comment:
TODO what should the base class be when there is no qobject macro ? Should it just be empty ?